### PR TITLE
Adding page of notebook summary

### PR DESCRIPTION
**Pull request by MatheusClaudi**

### CODE REVIEW

File: docs/api/api_scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# ML-Git Step-by-Step Guide
+
+## About
+
+In order to facilitate the learning process of using the ML-Git API, it's offered a series of step-by-step guides that emulate real situations in the learning context using the correct API command sequences to be used in each scenario.
```
Comment (Vieira-Carlos) [inline docs/api/api_scripts/README.md:5]:
```suggestion
In order to facilitate the learning process of using the ML-Git API, we offer a series of step-by-step guides that emulate real situations in the learning context applying the correct API command sequences that should be used in each scenario.
```

File: docs/api/api_scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+The guides were created using Jupyter notebooks, as their use offers the possibility to run step-by-step code snippets in a format similar to explanatory document, thus becoming ideal for teaching new users how to use api in real-use scenarios.
```
Comment (Vieira-Carlos) [inline docs/api/api_scripts/README.md:7]:
```suggestion
The guides were created using Jupyter notebooks, as they offer the possibility to run, step-by-step, code snippets in a format similar to an explanatory document, thus becoming ideal for teaching new users how to use the api in real case scenarios.
```

File: docs/api/api_scripts/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+### **How execute notebooks:**
+1. To run notebooks more easily, a docker environment has been created that performs all the environment settings required by the user. So make sure that the procedures in the [local environment configuration](https://github.com/HPInc/ml-git/tree/main/docker) section have been performed.
+    
+### **Summary of existing notebooks:**
+
+- [basic_flow](https://github.com/HPInc/ml-git/blob/main/docs/api/api_scripts/basic_flow.ipynb) - This notebook describes a basic flow of ml-git with api. There you will learn how to initialize an ML-Git project, how to perform all necessary configuration steps and how to version a dataset.<br/>
```
Comment (Vieira-Carlos) [inline docs/api/api_scripts/README.md:14]:
```suggestion
- [basic_flow](https://github.com/HPInc/ml-git/blob/main/docs/api/api_scripts/basic_flow.ipynb) - This notebook describes a basic execution flow of ml-git with its api. There, you will learn how to initialize an ML-Git project, how to perform all the necessary configuration steps and how to version a dataset.<br/>
```

File: docs/api/api_scripts/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+- [clone_repository](https://github.com/HPInc/ml-git/blob/main/docs/api/api_scripts/clone_repository.ipynb) - This notebook describes how clone a ML-Git repository.
```
Comment (Vieira-Carlos) [inline docs/api/api_scripts/README.md:15]:
```suggestion
- [clone_repository](https://github.com/HPInc/ml-git/blob/main/docs/api/api_scripts/clone_repository.ipynb) - This notebook describes how to clone an ML-Git repository.
```

File: docs/api/api_scripts/README.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+- [mnist_random_forest_api](https://github.com/HPInc/ml-git/blob/main/docs/api/api_scripts/mnist_notebook/mnist_random_forest_api.ipynb) - This notebook describes a basic flow with ml-git with api. In it, we will show how to obtain a dataset already versioned by ml-git, how to perform a versioning process of a model and new data generated, using MNIST dataset.
```
Comment (Vieira-Carlos) [inline docs/api/api_scripts/README.md:16]:
```suggestion
- [mnist_random_forest_api](https://github.com/HPInc/ml-git/blob/main/docs/api/api_scripts/mnist_notebook/mnist_random_forest_api.ipynb) - This notebook describes a basic execution flow of ml-git with its api. In it, we show how to obtain a dataset already versioned by ml-git, how to perform the versioning process of a model and the new data generated, using the MNIST dataset.
```

File: docs/api/api_scripts/README.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+- [mnist_random_forest_cli](https://github.com/HPInc/ml-git/blob/main/docs/api/api_scripts/mnist_notebook/mnist_random_forest_cli.ipynb) - This notebook describes a basic flow with ml-git. In it, we will show how to obtain a dataset already versioned by ml-git, how to perform a versioning process of a model and new data generated, using the MNIST dataset.
```
Comment (Vieira-Carlos) [inline docs/api/api_scripts/README.md:17]:
```suggestion
- [mnist_random_forest_cli](https://github.com/HPInc/ml-git/blob/main/docs/api/api_scripts/mnist_notebook/mnist_random_forest_cli.ipynb) - This notebook describes a basic execution flow with ml-git. In it, we show how to obtain a dataset already versioned by ml-git, how to perform the versioning process of a model and the new data generated, using the MNIST dataset.
```

File: docs/api/api_scripts/README.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+- [checkout_with_sample](https://github.com/HPInc/ml-git/blob/main/docs/api/api_scripts/multiple_datasets_notebook/checkout_with_sample.ipynb) - This notebook describes how perform a checkout in ML-Git using samples of dataset.
```
Comment (Vieira-Carlos) [inline docs/api/api_scripts/README.md:18]:
```suggestion
- [checkout_with_sample](https://github.com/HPInc/ml-git/blob/main/docs/api/api_scripts/multiple_datasets_notebook/checkout_with_sample.ipynb) - This notebook describes how to perform a checkout operation with ML-Git using samples of a dataset.
```

File: docs/api/api_scripts/README.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+- [multiple_datasets](https://github.com/HPInc/ml-git/blob/main/docs/api/api_scripts/multiple_datasets_notebook/multiple_datasets.ipynb) - This notebook describes how to handle scenario when the same file is present in more than one dataset.
```
Comment (Vieira-Carlos) [inline docs/api/api_scripts/README.md:19]:
```suggestion
- [multiple_datasets](https://github.com/HPInc/ml-git/blob/main/docs/api/api_scripts/multiple_datasets_notebook/multiple_datasets.ipynb) - This notebook describes how to handle the scenario where the same file is present in more than one dataset.
```

**Review by Vieira-Carlos — CHANGES_REQUESTED** [review]
A few suggestions. Also, it would be good to do the same changes for the notebook's descriptions in each of their files.

**Review by Vieira-Carlos — APPROVED** [review]

File: docs/api/api_scripts/README.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
+# ML-Git Step-by-Step Guide
+
+## About
+
+In order to facilitate the learning process of using the ML-Git API, we offer a series of step-by-step guides that emulate real situations in the learning context applying the correct API command sequences that should be used in each scenario.
```
Comment (andersonpablosilva) [inline docs/api/api_scripts/README.md:5]:
```suggestion
In order to facilitate the learning process of using the ML-Git API, we offer a series of step-by-step guides that emulate real situations in the learning context by applying the correct API command sequences that should be used in each scenario.
```

File: docs/api/api_scripts/README.md [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+
+The guides were created using Jupyter notebooks, as they offer the possibility to run, step-by-step, code snippets in a format similar to an explanatory document, thus becoming ideal for teaching new users how to use the api in real case scenarios.
```
Comment (andersonpablosilva) [inline docs/api/api_scripts/README.md:7]:
```suggestion
The guides were created using Jupyter notebooks, as they offer the possibility to run, step-by-step, code snippets in a format similar to an explanatory document, thus becoming ideal for teaching new users how to use the API in real case scenarios.
```

File: docs/api/api_scripts/README.md [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+
+### **How execute notebooks:**
+1. To run notebooks more easily, a docker environment has been created that performs all the environment settings required by the user. So make sure that the procedures in the [local environment configuration](https://github.com/HPInc/ml-git/tree/main/docker) section have been performed.
+    
+### **Summary of existing notebooks:**
+
+- [basic_flow](https://github.com/HPInc/ml-git/blob/main/docs/api/api_scripts/basic_flow.ipynb) - This notebook describes a basic execution flow of ml-git with its api. There, you will learn how to initialize an ML-Git project, how to perform all the necessary configuration steps and how to version a dataset.<br/>
```
Comment (andersonpablosilva) [inline docs/api/api_scripts/README.md:14]:
```suggestion
- [basic_flow](https://github.com/HPInc/ml-git/blob/main/docs/api/api_scripts/basic_flow.ipynb) - This notebook describes a basic execution flow of ml-git with its API. There, you will learn how to initialize an ML-Git project, how to perform all the necessary configuration steps and how to version a dataset.<br/>
```

File: docs/api/api_scripts/README.md [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+- [clone_repository](https://github.com/HPInc/ml-git/blob/main/docs/api/api_scripts/clone_repository.ipynb) - This notebook describes how to clone an ML-Git repository.
+- [mnist_random_forest_api](https://github.com/HPInc/ml-git/blob/main/docs/api/api_scripts/mnist_notebook/mnist_random_forest_api.ipynb) - This notebook describes a basic execution flow of ml-git with its api. In it, we show how to obtain a dataset already versioned by ml-git, how to perform the versioning process of a model and the new data generated, using the MNIST dataset.
```
Comment (andersonpablosilva) [inline docs/api/api_scripts/README.md:16]:
```suggestion
- [mnist_random_forest_api](https://github.com/HPInc/ml-git/blob/main/docs/api/api_scripts/mnist_notebook/mnist_random_forest_api.ipynb) - This notebook describes a basic execution flow of ml-git with its API. In it, we show how to obtain a dataset already versioned by ml-git, how to perform the versioning process of a model and the new data generated, using the MNIST dataset.
```

**Review by andersonpablosilva — COMMENTED** [review]
Just a few suggestions

**Review by andersonpablosilva — APPROVED** [review]

**Review by lucasncabral — APPROVED** [review]

**Pull request merged**